Submitty/Submitty
# [Bugfix:TAGrading] Show hidden points row to grader

### What is the current behavior?
fixes issue #6667 Currently when a grader is grading an assignment the autograding panel does not show the Autograding total with Hidden points unless the grades have been released.

### What is the new behavior?
All Graders EXCEPT PEER GRADERS can now see the row for autograding total with hidden points while grading. Students still can't see hidden points until grades are released.


### Other information?
To test:
Grade 2 assignments with hidden auto grading points, one where grading is still open and one where grades have been released, see that you can see the hidden points row no matter what.
